### ISSUE TITLE
Logo for CodeCTRL

### ISSUE THREAD

**Issue by STBoyden** [Authentura/codectrl#49]
Look into a design for a logo for CodeCTRL to be used on the website, GitHub repository, within the application itself and for the taskbar/dock icon.

I can have a talk with somebody to see if they can come up with some ideas, @svmorris could you also get in touch with somebody?

**Comment by immattdavison** [comment]
I would like to do this, please assign me

**Comment by STBoyden** [comment]
> I would like to do this, please assign me

Happily :) Thank you.

**Comment by avatar-10000** [comment]
@STBoyden I can help in creating a cool logo for CodeCTRL.
It is already assigned to @immattdavison, I would like to work together and help him as well.

**Comment by Anudeep-313** [comment]
I am good at creating logos, i can design few sample logos for codeCTRL if you like them assign me this issue.

**Comment by STBoyden** [comment]
Absolutely, I'll assign you guys later. Feel free to work in the meantime.

**Comment by Anudeep-313** [comment]
# I created few logos, Review them and let me know if they are good.
Would like know if y'all need any changes in the logo(colour palette).

# Logo1
![LOGO](https://user-images.githubusercontent.com/84098720/193564093-4f0f5561-8be2-407d-95e9-409ebc1a7be5.png)
---
# Logo2
![Logo1](https://user-images.githubusercontent.com/84098720/193564100-15571e3d-5aac-49be-8083-fd11d584c5ac.png)
---
# Logo3
![logo3](https://user-images.githubusercontent.com/84098720/193564105-89b26348-0fd8-4930-a4c4-01df0008fd8c.png)

**Comment by STBoyden** [comment]
@Anudeep-313 We'll send you my thoughts in a little while, currently busy with IRL stuff. Also waiting for the others to respond with their additions too.

Additionally, just some information for you guys, you will all receive credits for your contributions regardless of whether your logo is finally chosen or not on the README. I'll try and find a way for you guys to all get points towards your Hacktoberfest PRs/MRs but we'll see when it comes to a choice.

**Comment by Anudeep-313** [comment]
@STBoyden Thank you for the response and waiting for your thoughts on the designs.

**Comment by Anudeep-313** [comment]
> > @STBoyden Thank you for the response and waiting for your thoughts on the designs.
> 
> I'll try and get back to you ASAP.

great, thank you!

**Comment by STBoyden** [comment]
Apologies for taking so long.

@Anudeep-313. Personally, my preferred logos out of those 4 are the last two, especially the last one as it is more in-line with our colours, so thank you for that. 

With regard to the last one, would you mind using a font called "Readex" and using similar colours to our company's logo on the CodeCTRL website? The half-cog is a nice touch. Would it be possible to add some depth to it, like you had done with the logo prior?

**Comment by Anudeep-313** [comment]
I've done the changes, please take a look.

- 250x250px

![CodeCTRL_LOGO](https://user-images.githubusercontent.com/84098720/194130355-fd3a0a70-12d8-4455-acb8-23b4f5742554.png)
---
- 500x500px

![CodeCTRL_Logo1](https://user-images.githubusercontent.com/84098720/194130816-41967a52-d64c-436f-87f9-4a18def612f8.png)

**Comment by Anudeep-313** [comment]
- with white BG colour.

![CodeCTRl_Whitebg](https://user-images.githubusercontent.com/84098720/194131425-d20b40ee-1c35-4f17-ac45-8a0eaab94679.png)

**Comment by Anudeep-313** [comment]
Let me know if its good to create the PR. 😃

**Comment by Anudeep-313** [comment]
Hi @STBoyden, PTAL at the new updated logos.

**Comment by STBoyden** [comment]
> Hi @STBoyden, PTAL at the new updated logos. 

Apologies, I hadn't received a GH notification. These look amazing. Give me a bit to discuss with the others and if all goes well, you can go ahead and create a PR.

**Comment by Anudeep-313** [comment]
> > Hi @STBoyden, PTAL at the new updated logos.
> 
> Apologies, I hadn't received a GH notification. These look amazing. Give me a bit to discuss with the others and if all goes well, you can go ahead and create a PR.

Alright no worries, Thank you!!

**Comment by svmorris** [comment]
Hi @Anudeep-313 (also an organisation member here)
I really like your logo concepts and I think they could definitely work for us. However, there are just some small changes that would like to suggest. Here are some of my thoughts in no particular order:

First of all, I love the shadows on the last logo, but not the orange tint on the left side of the gear. I am not sure how it would look without it, but would it be possible to try that? Alternatively, I love the simplicity of logo 4. Could we see how that would look with the text of the last logo?

Secondly, I am slightly worried about how this will fit on a webpage, especially if it is slightly tight on space. The large gear makes the text by comparison be really small. For example: imagine fitting this logo into the header of a website, the actual bit that says "codeCTRL" would end up looking very small and hard to read. Have you tried experimenting with using the gear as part of the text, or just changing the difference in size?

Thirdly, I know this goes slightly against what my colleague said, but I think the logo looks better with the pink text and a grey or black gear. 

Lastly, Just wanted to say thank you for this work you have done. Even if you cannot make these changes, we will most likely use one of these logos as I think the gears are an amazing idea. Thank you for your great work!

**Comment by Anudeep-313** [comment]
This logo in the header of the website will for sure make the text "CodeCTRL" look small so i'll try fixing that and also update/design few logos with your desired changes and get back.

**Comment by svmorris** [comment]
Thank you mate, you are a legend. Amazing work!

**Comment by Anudeep-313** [comment]
# I've made the changes, PTAL!!

# 1

![codectrl1](https://user-images.githubusercontent.com/84098720/195143642-5dbbc812-8c61-414c-abda-468d2c7ae357.png)

![codectrl1blackbg](https://user-images.githubusercontent.com/84098720/195143671-d923e9cf-9eda-4a7e-a39e-10f45a8745ec.png)
---

# 2

![Codectrl2](https://user-images.githubusercontent.com/84098720/195143787-9cc070ae-9b0e-4eab-877e-dbd9d2f7098f.png)

![Codectrl-2 blackbg](https://user-images.githubusercontent.com/84098720/195143801-6eb3b583-0340-4f67-a569-bc7bbc0b0525.png)

---

# 3

![Codectrl-greygear](https://user-images.githubusercontent.com/84098720/195143884-70be1bb1-e57c-47e5-a1f9-b730977d07bd.png)

**Comment by Anudeep-313** [comment]
> # I've made the changes, PTAL!!
> # 1
> ![codectrl1](https://user-images.githubusercontent.com/84098720/195143642-5dbbc812-8c61-414c-abda-468d2c7ae357.png)
> 
> ## ![codectrl1blackbg](https://user-images.githubusercontent.com/84098720/195143671-d923e9cf-9eda-4a7e-a39e-10f45a8745ec.png)
> # 2
> ![Codectrl2](https://user-images.githubusercontent.com/84098720/195143787-9cc070ae-9b0e-4eab-877e-dbd9d2f7098f.png)
> 
> ![Codectrl-2 blackbg](https://user-images.githubusercontent.com/84098720/195143801-6eb3b583-0340-4f67-a569-bc7bbc0b0525.png)
> 
> # 3
> ![Codectrl-greygear](https://user-images.githubusercontent.com/84098720/195143884-70be1bb1-e57c-47e5-a1f9-b730977d07bd.png)

@svmorris let me know your thoughts about the new changes made to the logos.

**Comment by Anudeep-313** [comment]
@svmorris let me know your thoughts about the new changes made to the logos.

**Comment by svmorris** [comment]
@Anudeep-313 Terribly sorry for the late reply.
I love both 1 and 2 a lot, amazing work!

I will leave the final choice to our primary maintainer @STBoyden, as soon as he replies I would say you should go ahead with the PR. I messaged him privately to make sure he doesn't miss it.

Thank you again for the amazing work!

**Comment by STBoyden** [comment]
@Anudeep-313 Hey mate, apologies for the wait!

The logos look amazing, please go ahead and create a PR with all of those, if that's possible. Could you also include square logos of just the cogs as well? Also, if possible, could you provide all the logos (both the cog and the cog and text) in SVG format?

When you create the PR, make sure to put them in `/assets/logos`.

Otherwise, thank you very much. I will add your name to the README and the website as the designer of the logos, much appreciated :). If you would like, do you have anything we can link to, maybe a website or portfolio?

**Comment by STBoyden** [comment]
@Anudeep-313 Whenever you can, please create the PR :) Looking forward to merging it!

**Comment by Anudeep-313** [comment]
Great!!, I'll create separate logos of the cogs and create a PR.
I don't have any portfolio as such to link instead link my GitHub profile - [https://github.com/Anudeep-313](https://github.com/Anudeep-313)

**Comment by STBoyden** [comment]
Great, thank you :)

**Comment by Anudeep-313** [comment]
is it fine if i upload logos in png format ??

**Comment by STBoyden** [comment]
Yes, but would you also be able to provide them as SVGs?

**Comment by Anudeep-313** [comment]
> Yes, but would you also be able to provide them as SVGs?

- I rendered all my edits to png format and didn't save the psd file, Converting these png logos to svg format is reducing the resolution of the logos and there are few problem.
- I'll try figuring out a way to provide these logos in svg format as well. 

##  Y'all have all the permissions to use these logos freely.

# PTAL at #67
